### ISSUE TITLE
RtGistCommentTest.java:57-59: Apply this rule to all...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1399]
The puzzle `1394-59643c80` from #1394 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/485406ec4da1581ec962e943adaa729d2d226766/src/test/java/com/jcabi/github/RtGistCommentTest.java#L57-L59

The puzzle was created by George Aristy on 23-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#1399](https://github.com/jcabi/jcabi-github/issues/1399) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1399](https://github.com/jcabi/jcabi-github/issues/1399) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1131527228); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on review for PR #1402

**Comment by 0crat** [comment]
> @0crat waiting on review for PR #1402 ([here](https://github.com/jcabi/jcabi-github/issues/1399#issuecomment-385538589))

@llorllale The impediment for [#1399](https://github.com/jcabi/jcabi-github/issues/1399) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1394-59643c80` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1404](https://github.com/jcabi/jcabi-github/issues/1404) is still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1399](https://github.com/jcabi/jcabi-github/issues/1399) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the only puzzle [#1404](https://github.com/jcabi/jcabi-github/issues/1404) is solved here.
